tests/python/23_bandwidth/23_bandwidth.py

```diff
@@ -52,7 +52,7 @@ def getThreshold(devHandle):
         threshold = 20000
     if b"u2x4" in deviceInfo.mName or b"U2x4" in deviceInfo.mName or b"u2_gen3x4" in deviceInfo.mName:
         threshold = 10000
-    if b"_u25_" in deviceInfo.mName: # so that it doesn't set theshold for u250
+    if b"_u25_" in deviceInfo.mName or b"_u30_" in deviceInfo.mName: # so that it doesn't set theshold for u250
```

Comment:
I know why this code is here.  Please please please, post 2020.2 lets find a better way of doing this.

Reply:
Yes. This is the reason why we are introducing platform.json, to make it more data driven